arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -755,6 +772,43 @@ replication doing a resync, however.)");
               arangodb::options::Flags::OnSingle))
       .setIntroducedIn(31005);
 
+  options
+      ->addOption("--cache.min-value-size-for-edge-compression",
+                  "The size threshold (in bytes) from which on payloads in the "
+                  "edge index cache transparently get LZ4-compressed.",
+                  new SizeTParameter(&_minValueSizeForEdgeCompression, 1, 0,
+                                     1073741824ULL),
+                  arangodb::options::makeFlags(
+                      arangodb::options::Flags::DefaultNoComponents,
+                      arangodb::options::Flags::OnDBServer,
+                      arangodb::options::Flags::OnSingle))
+      .setLongDescription(
+          R"(By transparently compressing values in the in-memory
+edge index cache, more data can be held in memory than without compression.  
+Storing compressed values can increase CPU usage for the on-the-fly compression 
+and decompression. In case compression is undesired, this option can be set to a 
+very high value, which will effectively disable it. To use compression, set the
+option to a value that is lower than medium-to-large average payload sizes.
+It is normally not that useful to compress values that are smaller than 100 bytes.)")
+      .setIntroducedIn(31102);
+
+  options
+      ->addOption(
+          "--cache.acceleration-factor-for-edge-compression",
+          "The acceleration factor for the LZ4 compression of in-memory "
+          "edge cache entries.",
+          new UInt32Parameter(&_accelerationFactorForEdgeCompression, 1, 1, 10),
```

Comment:
Is 10 for this function mean max value? In such case it's incorrect, exist valid and useful examples with 50.
https://github.com/inikep/lzbench/blob/master/lz4/lz4.h#L190 correct max

Also I think important to note that 1 step increase compression speed commonly on 3%, and could increase a little decompress speed.
Or just point that lz4_compress_fast parameter